tests/lib/prepare-restore.sh

```diff
@@ -587,7 +587,9 @@ prepare_project() {
     # go mod runs as root and will leave strange permissions
     chown test:test -R "$SPREAD_PATH"
 
-    if [ "$BUILD_SNAPD_FROM_CURRENT" = true ]; then
+    if tests.info is-snapd-pkg-repo; then
```

Comment:
Could we have a comment here, something like:
```
# We are testing snapd snap on top of snapd from the archive
# of the tested distribution. Download snapd and snap-confine
# as they exist in the archive for further use.
```

Reply:
done